### ISSUE TITLE
data.umweltprofis.at: iCal service retired

### ISSUE THREAD

**Issue by andyboeh** [mampfes/hacs_waste_collection_schedule#414]
I had to change the interval in my data.umweltprofis.at waste schedule when I noticed that the server throws an error when requesting a new iCal link at https://data.umweltprofis.at/opendata/AppointmentService/index.aspx

I contacted their support and they told me that this service is not going to be fixed because they are currently working on (another?) app.

Should this integration be marked as broken for the time being? Existing iCal URLs continue to work, at least for the time being.

**Comment by BombusAlpinus** [comment]
@andyboeh i'm also having the same problem, wrote to them some weeks ago but didn't get an answer.
 did you manage to find a workaround? or do we have to wait?

**Comment by andyboeh** [comment]
Unfortunately, I don't know more than what I already wrote above. I'm not aware of any newer app either, but I didn't research that further.

At the same page where you request the iCal link you can also request data in XML format. The link contains a key and I'm unsure for how long this key is valid or if it ever changes. Parsing the XML data shouldn't be too difficult.

There is also the GEM2GO app that can display this schedule, but I did not further investigate where they get their data from.

**Comment by BombusAlpinus** [comment]
ok, i see... thanks for reply... maybe theres a solution with a self created calendar and wast collection schedule...

**Comment by BombusAlpinus** [comment]
hi, it looks like that it is working again... but... do i have to change something in the integration for using the xml api?

**Comment by BombusAlpinus** [comment]
Hi, i just tried to switch over to the xmlurl and it didn't work. 
Changed the URL from url: to xmlurl: and also to the path from point 6 as described. 
I also changed the value of the types according to the xml. 
E.g. from the old Type in the ics:
`customize:
        - type: "Abholung: Hausabfall"
          alias: hausabfall
          icon: mdi:delete`
to:
`      customize:
        - type: Hausabfall
          alias: hausabfall
          icon: mdi:delete`
which are visible in the XML link:
`<WasteType>Hausabfall</WasteType>`

but i get an error:
Unable to prepare setup for platform waste_collection_schedule.sensor: Unable to set up component.
09:59:05 – (FEHLER) setup.py - Die Nachricht ist zum ersten Mal um 09:59:05 aufgetreten und erscheint 4 mal
Error during setup of component waste_collection_schedule
09:59:05 – (FEHLER) waste_collection_schedule (benutzerdefinierte Integration)

thanks!
